src/serac/physics/boundary_conditions/boundary_condition.hpp

```diff
@@ -178,47 +178,27 @@ class BoundaryCondition {
 
   /**
    * @brief Projects the boundary condition over a field
+   * @param[in] time The time at which to project the boundary condition
    * @param[inout] state The field to project over
    */
```

Comment:
These doxygen comments are almost tautological-- can we improve the explanation of what these functions do? 

I'm not too familiar with these functions, is the following explanation accurate:

```cpp
  /**
   * @brief for each dof that is associated with this boundary condition, overwrite 
   * its value in `state` by evaluating the provided function at the node associated with that dof
   * @param[inout] state The vector of dofs to modify
   * @param[in] time The time at which to evaluate the function
   */
```